helm-charts/splunk-otel-collector/templates/deployment-cluster-receiver.yaml

```diff
@@ -77,10 +77,8 @@ spec:
         ` }}
         {{- $clusterReceiver.affinity | mustMergeOverwrite (fromYaml $clusterReceiverPodAntiAffinity) | toYaml | nindent 8 }}
       {{- end }}
-      {{- if $clusterReceiver.securityContext }}
       securityContext:
-        {{ toYaml $clusterReceiver.securityContext | nindent 8 }}
-      {{- end }}
+        {{- include "splunk-otel-collector.securityContext" (dict "isWindows" .Values.isWindows "securityContext" $clusterReceiver.securityContext) | nindent 8 }}
```

Comment:
We don't need to set `runAsUser: 0` in the gateway and the cluster receiver. Only on the agent, both otel col and fluentd.

Reply:
I added another parameter to the helper func , whether to set runAsUser or not .
This was the helper fun can be called to clean up runAsUser from windows instances for the gw and cluster receiver.